sillsdev/web-languageforge
# update PHP dependencies and remove communication feature

## Description

This PR seeks to update our PHP deps.  While in the process of upgrading, it was noticed that our SMS feature was never rolled out and was keeping the update from continuing without further research.  It was decided that we could simply remove all the SMS functionality.

### Type of Change

- dependency update and residual unused features

## Screenshots

### SMS feature before

<img width="767" alt="image" src="https://user-images.githubusercontent.com/4412848/203598780-752e8674-64d1-470f-988f-e3f835417a37.png">

### SMS feature after

<img width="708" alt="image" src="https://user-images.githubusercontent.com/4412848/203599857-da72cb8e-d16a-47ba-80fc-c6bf9bc1bf08.png">

## Checklist

- [x]  rebase off `develop` after #1610 is merged

## How to test

- verify automated tests passed
- general smoke test of app
- create an account to ensure emails are unaffected
- ensure communication preference section is gone from "My Profile"

## qa.languageforge.org testing

Testers should add his/her findings to end of the PR in a comment and include screenshots, files, etc that are beneficial.
